spark/src/main/scala/org/apache/spark/sql/delta/stats/StatisticsCollection.scala

```diff
@@ -459,16 +459,22 @@ object StatisticsCollection extends DeltaCommand {
    * @param name The name of the data skipping column for validating data type.
    * @param dataType The data type of the data skipping column.
    * @param columnPaths The column paths of all valid fields.
+   * @param insideStruct Tracks if we are inside a struct, in which case all types are valid
```

Comment:
"we are" -> "The data skipping column that we are validating is inside a struct"

Reply:
I think we should specify why "all types are valid" here, it would be confusing if an array is invalid but struct<array> is not.

The goal here is to collect stats as much as possible for only the **valid** types inside a struct if I'm not wrong.

Reply:
Added more to the param doc to try to clarify more